homeassistant/components/hassio/issues.py

```diff
@@ -61,25 +61,27 @@
 
 UNSUPPORTED_REASONS = {
     "apparmor",
+    "cgroup_version",
     "connectivity_check",
     "content_trust",
     "dbus",
     "dns_server",
     "docker_configuration",
     "docker_version",
-    "cgroup_version",
     "job_conditions",
     "lxc",
     "network_manager",
     "os",
     "os_agent",
+    "os_version",
     "restart_policy",
     "software",
     "source_mods",
     "supervisor_version",
     "systemd",
     "systemd_journal",
     "systemd_resolved",
+    "virtualization_image",
```

Comment:
Is it possible to add tests to keep aligned with the library enum?

Reply:
Yes it's possible. However, the current version `0.3.1` of `aiohasupervisor` does not contain `os_version` yet :see_no_evil: . I opened a separate PR to bump to `0.3.2b0`: https://github.com/home-assistant/core/pull/150893. I can either rebase this one after that got merged, or add the test in a follow up PR.

Reply:
Actually, @mdegat01 had a better idea: We can get rid of the list here entirly, and use the enum by default (exclude the `UNSUPPORTED_SKIP_REPAIR`, which we still need to keep. Then we can add a test which mock resolution info for all unsupported reasons in the client library enum, and then check if they actually get properly created (means translations are present).

Let's do this in a follow up to this.

Reply:
@epenet FWIW, with #150919 the list is now taken from the library enum.